lib/opensusebasetest.pm

```diff
@@ -498,11 +498,17 @@ sub firewall {
 # of wait_boot, e.g. see `git grep -l reboot | xargs grep -L wait_boot`
 sub post_fail_hook {
     my ($self) = @_;
-    return unless $self->{in_wait_boot};
     # In case the system is stuck in shutting down or during boot up, press
     # 'esc' just in case the plymouth splash screen is shown and we can not
     # see any interesting console logs.
-    send_key 'esc';
+    if ($self->{in_wait_boot}) {
+        send_key 'esc';
+        save_screenshot;
+        return;
+    }
+    
+    select_console 'log-console';
```

Comment:
We have concerns if this thing will work with remote installations, e.g. ipmi. So will conduct more verification runs not to break post_fail_hooks.